WordPress/Classes/ViewRelated/Blog/My Site/MySiteViewController.swift

```diff
@@ -551,7 +543,8 @@ final class MySiteViewController: UIViewController, UIScrollViewDelegate, NoSite
 
     @objc
     func presentInterfaceForAddingNewSite() {
-        noSitesViewController.rootView.handleAddNewSiteButtonTapped()
+        wpAssertionFailure("invalid_state")
+//        noSitesViewController.rootView.handleAddNewSiteButtonTapped()
```

Comment:
This is weird because it's only used from https://github.com/wordpress-mobile/WordPress-iOS/blob/8e243ee8e843b91dbe5c1c17644861feda31d61c/WordPress/Classes/System/RootViewPresenter%2BEditorNavigation.swift#L16. 

I don't understand when this scenario might happen or why showing a "create site" flow was considered a good option. I'll need to dig a bit more into it tomorrow, but, so far, it seems it should be OK to remove.